commons/data_access_layer/cosmos_db.py

```diff
@@ -348,11 +358,11 @@ class CosmosDBDao(CRUDDao):
     def __init__(self, repository: CosmosDBRepository):
         self.repository = repository
 
-    def get_all(self, conditions: dict = None, **kwargs) -> list:
+    def get_all(self, only_active = True, conditions: dict = None, **kwargs) -> list:
```

Comment:
I think is not neccesary to put this value here(unless you plan to do something with it). Is already defined in the repository method.

Reply:
Ok, it has been adjusted, with a new approach to avoid the only_active argument.